unpackaged/config/platformencryption_fields/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@jofsky Is the thought here that these field should be marked as encrypted when we run this flow? If so, if we wanted all of them, we can just add to this?